specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+{
+  "swagger": "2.0",
+  "host": "management.azure.com",
+  "info": {
+    "version": "2024-01-01-preview",
+    "title": "Management Groups API",
+    "description": "The Azure Management Groups API enables consolidation of multiple \nsubscriptions/resources into an organizational hierarchy and centrally \nmanage access control, policies, alerting and reporting for those resources.\n This preview version will also include APIs for new resource type called Applications.\n"
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "ManagementGroups",
+      "description": "A Management Group is a customer defined scope (grouping mechanism) that \nprovides access control (authorization), policy management and reporting. \nManagement Groups are organized in a strictly tree-based hierarchy.\n"
+    },
+    {
+      "name": "Operations",
+      "description": "Management operations supported by the Microsoft.Management resource provider.\n"
+    },
+    {
+      "name": "Entities",
+      "description": "A list of entities that belong to the Management Groups.\n"
+    },
+    {
+      "name": "Applications",
+      "description": "An Application is a customer created scope, where multiple subscriptions or applications can be grouped by. \n"
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.Management/managementGroups": {
```

Comment:
>/providers/Microsoft.Management/managementGroups [](http://example.com/codeflow?start=5&length=48)

New tenant-level APIs require [email] signoff from PAS team. Email to SARBACRev@microsoft.com, prasnal@microsoft.com, ARM Auth Team <armrbac@microsoft.com>. Please get signoff covering all new tenant-level paths/APIs.